src/glossary.rst

```diff
@@ -1587,17 +1588,34 @@ variant]s`.
 :dp:`fls_o6ih6n1z1566`
 See :s:`EnumDeclaration`.
 
+.. _fls_H6aUAUjNlx6z:
+
+enum value
+^^^^^^^^^^
+
+:dp:`fls_QdBTdVLB2xHk`
+An :dt:`enum value` is a :t:`value` of an :t:`enum type`.
+
 .. _fls_klwlx5jixwud:
 
 enum variant
 ^^^^^^^^^^^^
 
 :dp:`fls_9jq4keg9y94u`
-An :dt:`enum variant` specifies a :t:`value` of an :t:`enum type`.
+An :dt:`enum variant` is a construct that declares one of the
+possible variations of an :t:`enum`.
 
 :dp:`fls_tj2s55onen6b`
 See :s:`EnumVariant`.
 
+.. _fls_mKxBWCojhnWu:
+
+enum variant value
+^^^^^^^^^^^^^^^^^^
+
+:dp:`fls_VQRqNPFFWmDp`
+An :dt:`enum variant value` is the :t:`enum value` of the corresponding :t:`enum` of the :t:`enum variant`.
```

Comment:
I didn't want to use that, because that discards the relationship to the enum the variant belongs to. Since any enum variant value is also an enum value of the enum the variant belongs to.

Reply:
I see, OK!